packages/common/src/operations.graphql

```diff
@@ -92,6 +92,39 @@ query invoice($id: String!) {
             note
             locationDescription
             sellPricePerPack
+            stockLine {
```

Comment:
I've added the stock line for the use case:

- Delete persisted line (so, flag and delete later)
- Create line for the same stock line

So in this case, reuse the 'deleted' line, which is easier if we have the stock line id (which I was already matching with, wrongly, as we didn't have the stock line)